test/fixture/generatedTypeScript/interfaces/IElement.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable import/prefer-default-export, import/no-cycle, @typescript-eslint/naming-convention, @typescript-eslint/no-empty-interface */
+import { 
+  IExtension,
+  IType,
+} from "../internal";
+
+export interface IElement extends IType {
```

Comment:
Codacy found an issue: [Interface name must not be prefixed with "I".](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5923420)